apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -131,7 +176,7 @@ export default class ScriptEditor extends React.Component {
 
   presubmit = e => {
     const videoKeysBefore = (
-      this.props.lessonLevelData.match(VIDEO_KEY_REGEX) || []
+      this.state.lessonLevelData.match(VIDEO_KEY_REGEX) || []
```

Comment:
I'm not 100% sure how to confirm this, but it seems like this code is trying to compare against the `lessonLevelData` that was sent down from the server, in which case i think you might want to look at `this.props.initialLessonLevelData` here instead?

Reply:
Interesting. I know the wrap up video is only used by some of the very old hour of code courses and I think this has to do with that. I will trust your assessment here and update to the initialLessonLevelData